test/src/edu/stanford/nlp/util/AcronymMatcherTest.java

```diff
@@ -19,6 +20,14 @@ public void testBasic() {
     assertTrue(AcronymMatcher.isAcronym("ECC", "Election Complaints Commission".split("\\s+")));
   }
 
+  @Test
+  public void testBasicWithBoundaries(){
+    assertTrue(AcronymMatcher.isAcronym("B", "B".split("\\s+")));
+    assertFalse(AcronymMatcher.isAcronym("", "B".split("\\s+")));
+//  assertFalse(AcronymMatcher.isAcronym("", "A".split("\\s+")));
```

Comment:
@RinorSefa  is this also a failing test?

Reply:
Yes I believe that's the other bug he found